doc/gendocs.sh

```diff
@@ -3,5 +3,11 @@ set -xeuo pipefail
 rm -rf _build
 rm -rf apidoc
 sphinx-apidoc -T -e -o apidoc ../src/commissaire/
-# TODO: Add the other repos here to get full api docs
+if [ -d "../../commissaire-service/" ]; then
+  sphinx-apidoc -T -e -o apidoc ../../commissaire-service/src/commissaire_service/
+fi
+if [ -d "../../commissaire-http/" ]; then
```

Comment:
It is looking outside. However, it isn't required. If the repos exist then it will regenerate the api docs. If they do not it will skip them. I will still put a note in there though. I think that's a good idea just in case.